### PR TITLE
[RELEASE] 0.16.13

### PR DESCRIPTION
**Pull request by wookasz**
release prep for 0.16.13

### CODE REVIEW

File: docs/docusaurus/docs/changelog.md [hunk 1]
```diff
@@ -10,6 +10,58 @@ title: Changelog
...
+* [DOCS] removes block-config docs for source data systems in the cloud ([#7871](https://github.com/great-expectations/great_expectations/pull/7871))
+* [DOCS] Continuing CLI Update ([#7876](https://github.com/great-expectations/great_expectations/pull/7876))
+* [DOCS] Removes the SQL block config guides. ([#7886](https://github.com/great-expectations/great_expectations/pull/7886))
+* [DOCS] Remove jq dependency for building docs ([#7893](https://github.com/great-expectations/great_expectations/pull/7893))
+* [DOCS] Less verbose logging during docs build ([#7894](https://github.com/great-expectations/great_expectations/pull/7894))
+* [DOCS] Corrections to Document "How to request data from a Data Asset" with proper FDS implementation and integration test. ([#7896](https://github.com/great-expectations/great_expectations/pull/7896))
+* [DOCS] Remove or Modify References to the CLI ([#7875](https://github.com/great-expectations/great_expectations/pull/7875))
+* [DOCS] Updated contribution docs ([#7880](https://github.com/great-expectations/great_expectations/pull/7880)) (thanks @TrangPham)
```
Comment (Kilo59) [inline docs/docusaurus/docs/changelog.md]:
```suggestion
* [DOCS] Updated contribution docs ([#7880](https://github.com/great-expectations/great_expectations/pull/7880))
```

File: docs/docusaurus/docs/changelog.md [hunk 2]
```diff
@@ -10,6 +10,58 @@ title: Changelog
...
+* [FEATURE] : DataProfilerStructuredDataAssistant Float Rule ([#7842](https://github.com/great-expectations/great_expectations/pull/7842)) (thanks @micdavis)
+* [FEATURE] DataProfilerColumnDomainBuilder ([#7920](https://github.com/great-expectations/great_expectations/pull/7920)) (thanks @micdavis)
+* [FEATURE] Add "batch.columns()" convenience method to Fluent DataAsset implementation. ([#7926](https://github.com/great-expectations/great_expectations/pull/7926))
+* [BUGFIX] Fix inability to extend SimpleCheckpoint -- and several additional enhancements and clean up ([#7879](https://github.com/great-expectations/great_expectations/pull/7879))
+* [BUGFIX] Delete ExpectationSuite by name in GX Cloud ([#7881](https://github.com/great-expectations/great_expectations/pull/7881))
+* [BUGFIX] optional dataframe on datasources ([#7862](https://github.com/great-expectations/great_expectations/pull/7862))
+* [BUGFIX] DX-506: Fix sparkDF cannot compute mean for DecimalType ([#7867](https://github.com/great-expectations/great_expectations/pull/7867)) (thanks @TrangPham)
```
Comment (Kilo59) [inline docs/docusaurus/docs/changelog.md]:
```suggestion
* [BUGFIX] DX-506: Fix sparkDF cannot compute mean for DecimalType ([#7867](https://github.com/great-expectations/great_expectations/pull/7867))
```

File: docs/docusaurus/docs/changelog.md [hunk 3]
```diff
@@ -10,6 +10,58 @@ title: Changelog
...
+* [MAINTENANCE] partial `checkpoint` type checking ([#6914](https://github.com/great-expectations/great_expectations/pull/6914))
+* [MAINTENANCE] type-checking `checkpoint/actions.py` ([#7899](https://github.com/great-expectations/great_expectations/pull/7899))
+* [MAINTENANCE] Trigger docs_integration on develop ([#7902](https://github.com/great-expectations/great_expectations/pull/7902))
+* [MAINTENANCE] Finish type-checking `checkpoint` ([#7901](https://github.com/great-expectations/great_expectations/pull/7901))
+* [MAINTENANCE] Update How to set up GX to work with data in Azure Blob Storage ([#7910](https://github.com/great-expectations/great_expectations/pull/7910))
+* [MAINTENANCE] `CloudDataContext` `ExpectationsStore` can only request one Suite at a time ([#7905](https://github.com/great-expectations/great_expectations/pull/7905))
+* [MAINTENANCE] Removing engine-specific tests that assert generic behavior ([#7918](https://github.com/great-expectations/great_expectations/pull/7918))
+* [MAINTENANCE] Use with open file to reduce open files within tests ([#7906](https://github.com/great-expectations/great_expectations/pull/7906)) (thanks @TrangPham)
```
Comment (NathanFarmer) [inline docs/docusaurus/docs/changelog.md]: Did you also mean to remove this? Is this an internal user?

File: docs_rtd/changelog.rst [hunk 4]
```diff
@@ -4,6 +4,59 @@
...
+* [MAINTENANCE] partial `checkpoint` type checking ([#6914](https://github.com/great-expectations/great_expectations/pull/6914))
+* [MAINTENANCE] type-checking `checkpoint/actions.py` ([#7899](https://github.com/great-expectations/great_expectations/pull/7899))
+* [MAINTENANCE] Trigger docs_integration on develop ([#7902](https://github.com/great-expectations/great_expectations/pull/7902))
+* [MAINTENANCE] Finish type-checking `checkpoint` ([#7901](https://github.com/great-expectations/great_expectations/pull/7901))
+* [MAINTENANCE] Update How to set up GX to work with data in Azure Blob Storage ([#7910](https://github.com/great-expectations/great_expectations/pull/7910))
+* [MAINTENANCE] `CloudDataContext` `ExpectationsStore` can only request one Suite at a time ([#7905](https://github.com/great-expectations/great_expectations/pull/7905))
+* [MAINTENANCE] Removing engine-specific tests that assert generic behavior ([#7918](https://github.com/great-expectations/great_expectations/pull/7918))
+* [MAINTENANCE] Use with open file to reduce open files within tests ([#7906](https://github.com/great-expectations/great_expectations/pull/7906)) (thanks @TrangPham)
```
Comment (NathanFarmer) [inline docs_rtd/changelog.rst]: Did you also mean to remove this? Is this an internal user?

**Review by NathanFarmer — CHANGES_REQUESTED** [review]

File: docs/docusaurus/docs/changelog.md (hunk 3 above)
Comment (wookasz) [reply docs/docusaurus/docs/changelog.md]: thanks

File: docs_rtd/changelog.rst (hunk 4 above)
Comment (wookasz) [reply docs_rtd/changelog.rst]: thanks

**Review by Kilo59 — APPROVED** [review]

**Review by Kilo59 — APPROVED** [review]

**Review by NathanFarmer — APPROVED** [review]

**Pull request merged**